app/main.py

```diff
@@ -408,6 +412,7 @@ def update_live_alerts_data(
             # - the storage component that contains the dictionary with detection frame URLs;
             # - the storage component that serves as source of truth for the list of already loaded alerts;
             # - the HTML div that stores the list of sites where an alert is live
+            print(live_alerts, dict_images_url_live_alerts, new_loaded_frames, sites_with_live_alerts)
```

Comment:
I think that's for debug, careful to remove those when cleaning the PR :+1:

Reply:
Yes, thanks for that

Reply:
Done